playground/backend/containers/go/build.gradle

```diff
@@ -83,19 +84,22 @@ task copyDockerfileDependencies(type: Copy) {
 }
 
 docker {
-  name containerImageName(
-          name: project.docker_image_default_repo_prefix + "playground-backend-go",
-          root: project.rootProject.hasProperty(["docker-repository-root"]) ?
-                  project.rootProject["docker-repository-root"] :
-                  project.docker_image_default_repo_root)
-  files "./build/"
-  tags containerImageTags()
-  buildArgs(['BASE_IMAGE': project.rootProject.hasProperty(["base-image"]) ?
-                           project.rootProject["base-image"] :
-                           "golang:1.18-bullseye",
-             'SDK_TAG': project.rootProject.hasProperty(["sdk-tag"]) ?
-                  project.rootProject["sdk-tag"] : project.rootProject.sdk_version,
-             'SDK_TAG_LOCAL': project.rootProject.sdk_version])
+   name containerImageName(
+         name: project.docker_image_default_repo_prefix + "playground-backend-go",
+         root: project.rootProject.hasProperty(["docker-repository-root"]) ?
+               project.rootProject["docker-repository-root"] :
+               project.docker_image_default_repo_root)
+   files "./build/"
+   tags containerImageTags()
+   buildArgs(
+         ['BASE_IMAGE'   : project.rootProject.hasProperty(["base-image"]) ?
+               project.rootProject["base-image"] :
+               "golang:1.18-bullseye",
+          'SDK_TAG'      : project.rootProject.hasProperty(["sdk-tag"]) ?
+                project.rootProject["sdk-tag"] : project.rootProject.sdk_version,
+          'SDK_TAG_LOCAL': project.rootProject.sdk_version,
+          'GIT_COMMIT'   : getGitCommitHash(),
+          'GIT_TIMESTAMP': getGitCommitTimestamp()])
```

Comment:
The risk of wrapping more and more commands with gradle, the difficult it will be to troubleshoot in the future.

Reply:
IMO this is ok in order to get the reusability